app/views/tweets/edit_tweet.erb

```diff
@@ -0,0 +1,11 @@
+<% if @tweet.user_id != current_user.id %>
+<%= @tweet.content %><br>
+<% end %>
+
+<% if @tweet.user_id == current_user.id %>
```

Comment:
Ah I see. This clarifies why you didn't perform the user check in the controller. However, I would still include it there, in case a malicious user attempts to bypass your form.
Also, rather than just display the content for a user who doesn't own the tweet, I would redirect them to the show_tweet view (so that the URL remains accurate).